pom.xml

```diff
@@ -22,7 +22,10 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 	</properties>
-
+<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Comment:
review comments

Reply:
resolve

Reply:
reply to 139, I am 153

Reply:
@13911899075-han sdfdf

Reply:
@15301062826 @13911899075 @13911899075-han nizenmekan